test-project/Assets/EditmodeTests/Subscriptions/RequireablesUnlinkTests.cs

```diff
@@ -0,0 +1,114 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Subscriptions;
+using Improbable.Gdk.Test;
+using Improbable.Gdk.TestBases;
+using Improbable.Worker.CInterop;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace Improbable.Gdk.EditmodeTests.Subscriptions
+{
+    public class RequireablesUnlinkTests : SubscriptionsTestBase
+    {
+        private const long EntityId = 100;
+        private GameObject createdGameObject;
+
+        [SetUp]
+        public override void Setup()
+        {
+            base.Setup();
+            var template = new EntityTemplate();
+            template.AddComponent(new Position.Snapshot(), "worker");
+            template.AddComponent(new TestCommands.Snapshot(), "worker");
+            ConnectionHandler.CreateEntity(EntityId, template);
+            Update();
+        }
+
+        [TearDown]
+        public override void TearDown()
+        {
+            base.TearDown();
+            Object.DestroyImmediate(createdGameObject);
+        }
+
+        [Test]
+        public void Reader_is_disabled_if_component_removed()
```

Comment:
Should this be `if entity removed` or unlinked? The component itself is not removed. (How will this work if the QBI query removes the component from checkout?)

Reply:
These tests aren't concerned with ops coming down the wire (see https://github.com/spatialos/gdk-for-unity/commit/cb5319123d9b9332b4ffca51adde9edf8ee75be1 for those ones)